packages/fast-check/src/arbitrary/_internals/implementations/SchedulerImplem.ts

```diff
@@ -224,7 +224,7 @@ export class SchedulerImplem<TMetaData> implements Scheduler<TMetaData> {
 
   async waitOne(customAct?: SchedulerAct): Promise<void> {
     const waitAct = customAct || defaultSchedulerAct;
-    await this.act(() => waitAct(async () => await this.internalWaitOne()));
+    await this.act(() => waitAct(() => this.internalWaitOne()));
```

Comment:
I kept one extra await for now. I hope it could pass the tests with only one of them remaining. Anyway, the ideal thing would be to have none of them but so far it means breaking some tests so better investigating them before attempting anything dropping the last one.

By the way, before merging I should probably check how breaking such change could be for our users. The fact that dropping all the await broke our own tests proves that we should be careful with that change.